app/assets/index.html

```diff
@@ -24,12 +24,8 @@
   <body>
     <img class='title' src="/images/brigade-congress-2020.png"/>
     <section class='intro'>
-      <p>Welcome to this year's brigade congress! See below for helpful guides and the other tabs above!</p>
-      <h3>Helpful Guides:</h3>
-      <ul class='guides'>
-        <li><a>Guide to using Qiqochat</a></li>
-        <li><a>Guide to using Zoom</a></li>
-      </ul>
+      <p>Welcome to this year's brigade congress! See below for the Attendee Guide and checkout the other tabs above!</p>
+      <button class='button' href='/'>Attendee Guide</button>
```

Comment:
Let's make that button actually go somewhere if it's going to be in the code. Also, it should get styles by default from the CSS with the class "button button-primary"